lib/ansible/modules/database/postgresql/postgresql_sequence.py

```diff
@@ -0,0 +1,630 @@
+extends_documentation_fragment: postgres
+'''
+
+EXAMPLES = r'''
+- name: Create an ascending bigint sequence called foobar in the default
+        database
+  postgresql_sequence:
+    name: foobar
+
+- name: Create an ascending integer sequence called foobar, starting at 101
+  postgresql_sequence:
+    name: foobar
+    data_type: integer
+    start: 101
+
+- name: Create an descending sequence called foobar, starting at 101 and
+        preallocated 10 sequence numbers in cache
+  postgresql_sequence:
+    name: foobar
+    increment: -1
+    cache: 10
+    start: 101
+
+- name: Create an ascending sequence called foobar, which cycle between 1 to 10
+  postgresql_sequence:
+    name: foobar
+    cycle: yes
+    min: 1
+    max: 10
+
+- name: Create an ascending bigint sequence called foobar in the default
+        database with owner foobar
+  postgresql_sequence:
+    name: foobar
+    owner: foobar
+
+- name: Rename an existing sequence named foo to bar
+  postgresql_sequence:
+    name: foo
+    rename_to: bar
+
+- name: Change the schema of an existing sequence to foobar
+  postgresql_sequence:
+    name: foobar
+    newschema: foobar
+
+- name: Change the owner of an existing sequence to foobar
+  postgresql_sequence:
+    name: foobar
+    owner: foobar
+
+- name: Drop a sequence called foobar
+  postgresql_sequence:
+    name: foobar
+    state: absent
+
+- name: Drop a sequence called foobar with cascade
+  postgresql_sequence:
+    name: foobar
+    cascade: yes
+    state: absent
+'''
+
+RETURN = r'''
+state:
+  description: Sequence state at the end of execution.
+  returned: always
+  type: str
+  sample: 'present'
+sequence:
+  description: Sequence name.
+  returned: always
+  type: str
+  sample: 'foobar'
+queries:
+    description: List of queries that was tried to be executed.
+    returned: always
+    type: str
+    sample: [ "CREATE SEQUENCE \"foo\"" ]
+schema:
+    description: Name of the schema of the sequence
+    returned: always
+    type: str
+    sample: 'foo'
+data_type:
+    description: Shows the current data type of the sequence.
+    returned: always
+    type: str
+    sample: 'bigint'
+increment:
+    description: The value of increment of the sequence. A positive value will
+                 make an ascending sequence, a negative one a descending
+                 sequence.
+    returned: always
+    type: int
+    sample: '-1'
+minvalue:
+    description: The value of minvalue of the sequence.
+    returned: always
+    type: int
+    sample: '1'
+maxvalue:
+    description: The value of maxvalue of the sequence.
+    returned: always
+    type: int
+    sample: '9223372036854775807'
+start:
+    description: The value of start of the sequence.
+    returned: always
+    type: int
+    sample: '12'
+cycle:
+    description: Shows if the sequence cycle or not.
+    returned: always
+    type: str
+    sample: 'NO'
+owner:
+    description: Shows the current owner of the sequence.
```

Comment:
Does it mean the owner in the end of playbook runtime ? I suggest describing this